reco_utils/recommender/newsrec/IO/iterator.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import tensorflow as tf
+import abc
+
+
+class BaseIterator(object):
```

Comment:
Can we reuse the iterator from `deeprec`?

Reply:
Yes, I will make a PR.

Reply:
Just directly merge the new PR to this one. We don't need to create a separate one.